### PR TITLE
hbase wip

### PR DESCRIPTION
**Pull request by anthonyccri**

### CODE REVIEW

File: geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/collection/CloseableIterator.scala [hunk 1]
```diff
@@ -1,17 +1,13 @@
-/***********************************************************************
-* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
-* All rights reserved. This program and the accompanying materials
-* are made available under the terms of the Apache License, Version 2.0
-* which accompanies this distribution and is available at
-* http://www.opensource.org/licenses/apache2.0.php.
-*************************************************************************/
+/*
+ * Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
```
Comment (jnh5y) [inline geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/collection/CloseableIterator.scala]: This'll fail mvn license:check, no?

**Comment by jnh5y** [comment]
Looking good so far!

File: geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/collection/CloseableIterator.scala (hunk 1 above)
Comment (elahrvivaz) [inline geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/collection/CloseableIterator.scala]: yeah, good catch

File: geomesa-index-api/src/main/scala/org/locationtech/geomesa/index/geotools/GeoMesaFeatureSource.scala [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+
+  override def getName: Name = getSchema.getName
+
+  override def getDataStore: DataStore = ds
+
+  override def getSupportedHints: java.util.Set[Key] = hints
+
+  override def getInfo: ResourceInfo = new DelegatingResourceInfo(this)
+
+  override def addFeatureListener(listener: FeatureListener): Unit = throw new NotImplementedError()
```
Comment (elahrvivaz) [inline geomesa-index-api/src/main/scala/org/locationtech/geomesa/index/geotools/GeoMesaFeatureSource.scala]: seems better to fail hard no?

**Pull request closed without merging**